scopes/docs/docs/overview/overview.tsx

```diff
@@ -2,9 +2,18 @@ import React, { useContext } from 'react';
 import { ComponentContext } from '@teambit/component';
 import { ComponentPreview } from '@teambit/preview.ui.component-preview';
 import { StatusMessageCard } from '@teambit/design.ui.surfaces.status-message-card';
+import { ComponentOverview, TitleBadge } from '@teambit/component.ui.component-meta';
+import { useFetchDocs } from '@teambit/component.ui.hooks.use-fetch-docs';
+import styles from './overview.module.scss';
 
-export function Overview() {
+export type OverviewProps = {
+  titleBadges: TitleBadge[];
+};
+
+export function Overview({ titleBadges }: OverviewProps) {
   const component = useContext(ComponentContext);
+  const { data } = useFetchDocs(component.id.toString());
```

Comment:
this seems redundant - check if we can add it to the existing component query (at `use-component-query.ts`, I believe)